packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -149,6 +149,9 @@ if [ -e %{running_semaphore} ]; then
 fi
 
 %changelog
+* Thu Jun 1 2017 Adnan Khan <adnkha@amazon.com> - 1.14.2-1
+- Cache Agent version 1.14.2
+- Add functionality for running agent with userns=host when Docker has userns-remap enabled
```

Comment:
I think we're missing an entry here and in the spec file for the docker `17.03.x-ce` upgrade.